projects/addon-editor/components/toolbar/toolbar-navigation-manager.directive.ts

```diff
@@ -65,22 +65,29 @@ export class TuiToolbarNavigationManagerDirective {
             return wrapper;
         }
 
-        const lookedInside = tuiGetClosestFocusable(wrapper, false, wrapper, false);
+        const lookedInside = tuiGetClosestFocusable({
+            initial: wrapper,
+            root: wrapper,
+            keyboard: false,
+        });
 
         return (
             lookedInside ||
-            tuiGetClosestFocusable(wrapper, true, this.elementRef.nativeElement, false)
+            tuiGetClosestFocusable({
```

Comment:
There's `true` for second argument here, but it's not reflected in the options object.

Reply:
previous: true